index.js

```diff
@@ -17,15 +17,21 @@ class Board {
     for (let i = 0; i < Board.LANE_SIZE; i++) {
       const p = new Pawn(this.game, this, "white", 1, i);
       this.#board[p.row][p.file] = p;
-      const pB = new Pawn(this.game, this, "Black", 6, i);
+      const pB = new Pawn(this.game, this, "black", 6, i);
       this.#board[pB.row][pB.file] = pB;
     }
     for (let i = 0; i < Board.LANE_SIZE; i += 7) {
       const r = new Rook(this.game, this, "white", 0, i);
       this.#board[r.row][r.file] = r;
-      const rB = new Rook(this.game, this, "Black", 7, i);
+      const rB = new Rook(this.game, this, "black", 7, i);
       this.#board[rB.row][rB.file] = rB;
     }
+    for (let i = 2; i < 6; i += 3) {
+      const b = new Bishop(this.game, this, "white", 0, i);
+      this.#board[b.row][b.file] = b;
+      const bB = new Bishop(this.game, this, "black", 7, i);
+      this.#board[bB.row][bB.file] = bB;
+    }
```

Comment:
This is getting a bit repetitive and I think we can separate concerns a bit better.
Recommendations:
1. Define a static property on each `Piece` subclass - `startingFiles` of type `number[]` (array of numbers)
2. Define a static `initialize` method on each `Piece` subclass that accepts a `Board` as input 
 and creates a new instance of the subclass and then places it on the board for each of the `startingFiles`.
 3. Define a static `deriveStartingRow` method that accepts a color and returns the row it should start on
If you've done this correctly, the above code block should become:
```suggestion
      Pawn.initialize(this);
      Rook.initialize(this);
      Bishop.initialize(this);
```